src/software/jetson_nano/rotary_encoder/RotaryEncoder.py

```diff
@@ -0,0 +1,123 @@
+import Jetson.GPIO as GPIO
+
+# Channel B and Channel A states
+STATE_0 = [0, 0]
+STATE_1 = [0, 1]
+STATE_2 = [1, 1]
+STATE_3 = [1, 0]
+
+CLOCKWISE = 1
+COUNTERCLOCKWISE = -1
+IDLE = 0
+
+
+class RotaryEncoder:
```

Comment:
it would be good to add a comment here explaining what inputs are passed into the constructor

Reply:
Added comment for the inputs to the class